doc/source/tune/faq.rst

```diff
@@ -603,6 +603,50 @@ Failing to do so would cause error messages like ``Error message (1): fatal erro
 For AWS set up, this involves adding an IamInstanceProfile configuration for worker nodes.
 Please :ref:`see here for more tips <aws-cluster-s3>`.
 
+How can I use the awscli or gsutil command line commands for syncing?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Some users reported to run into problems with the default pyarrow-based syncing.
+In this case, a custom syncer that invokes the respective command line tools
+for transferring files between nodes and cloud storage can be implemented.
```

Comment:
This makes me wonder "why not make this the default again?"

Reply:
It's also unreliable :-)